packages/tools/pie-icons-react/package.json

```diff
@@ -50,17 +50,20 @@
   "devDependencies": {
     "@babel/core": "7.23.3",
     "@babel/node": "7.20.7",
-    "@justeattakeaway/pie-icons": "4.9.2",
-    "@justeattakeaway/pie-icons-configs": "4.5.0",
+    "@babel/preset-react": "7.18.6",
     "@svgr/core": "6.4.0",
     "pascal-case": "3.1.2",
     "react": "18.2.0",
+    "react-test-renderer": "18.0.0",
     "rollup": "3.20.2",
     "rollup-plugin-typescript2": "0.34.1",
-    "typescript": "5.1.3"
+    "typescript": "5.3.2"
   },
   "dependencies": {
-    "@babel/preset-react": "7.18.6",
-    "react-test-renderer": "18.0.0"
+    "@justeattakeaway/pie-icons": "4.9.2",
```

Comment:
Was this necessary for the Lit upgrade? I think we should be cautious about adding additional changes like this to the PR. I would prefer we do these kinds of things separately to minimise risk

Reply:
yes, it was unfortunately, I will share the Slack thread with you for this in a bit.